scripts/build_depends_project_list.sh

```
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+FILES="$(cat)"
+
+SCRIPT_DIR=$(cd $(dirname $0); pwd)
```

Shellcheck (0.10.0)

[warning] SCRIPT_DIR appears unused. Verify use (or export if used externally).

(SC2034)